pages/overview/what-is-jsonschema.md

```diff
@@ -6,36 +6,287 @@ title: What is JSON Schema?
+
+* This JSON document provides a structured representation of an order, making it easy to exchange, store, or process the order information in various applications or systems.
+
+
+### Without JSON Schema
+
+When working with JSON data, it can quickly become complex and difficult to manage, especially when dealing with nested structures. Without a standardized schema, it becomes challenging to validate and enforce constraints on the data. 
+
+For example, 
+
+```
+# Without JSON Schema
+data = {
+    "product": {
+        "name": "Widget",
+        "price": 10.99,
+        "quantity": 5
+    }
+}
+
+# Performing basic validation
+if "product" in data and isinstance(data["product"], dict) and "name" in data["product"] and "price" in data["product"]:
+    print("Valid JSON object.")
+else:
+    print("Invalid JSON object.")
+```
+
+In the above code snippet, we are performing basic validation to check if the JSON object has the required fields. Since this is a relatively simpler data, this way of checking works for now. 
+
+To show the challenges of performing data validation without using JSON Schema, we can take this exmaple:
+```
+# Without JSON Schema
+data = {
+    "order": {
+        "order_id": "123456",
+        "customer_name": "John Doe",
+        "items": [
+            {
+                "product_id": "P001",
+                "name": "T-shirt",
+                "quantity": 2,
+                "price": 19.99
+            },
+            {
+                "product_id": "P002",
+                "name": "Jeans",
+                "quantity": 1,
+                "price": 49.99
+            }
+        ],
+        "shipping_address": {
+            "street": "123 Main St",
+            "city": "New York",
+            "state": "NY",
+            "postal_code": "10001"
+        },
+        "total_amount": 89.97,
+        "status": "pending"
+    }
+}
+
+# Performing basic validation
+if (
+    "order" in data
+    and isinstance(data["order"], dict)
+    and "order_id" in data["order"]
+    and "customer_name" in data["order"]
+    and "items" in data["order"] and isinstance(data["order"]["items"], list)
+    and "shipping_address" in data["order"] and isinstance(data["order"]["shipping_address"], dict)
+    and "total_amount" in data["order"]
+    and "status" in data["order"]
+):
+    print("Valid JSON object.")
+else:
+    print("Invalid JSON object.")
+
+```
+Now we are dealing with a complex JSON structure that represents an order. The basic validation logic checks whether the required fields exist in the JSON object. However, as the structure becomes more complex, the validation code becomes more complicated and prone to errors. Moreover, this approach lacks support for checking data types, handling nested structures, and enforcing specific constraints.
+
+
+### Harness the power of JSON Schema 
```

Comment:
Instead of the current sentence "Harness the power of JSON Schema" I'll suggest something like "JSON Schema to the rescue"